README.md

```diff
@@ -38,6 +38,6 @@ This is continuously deployed from master [by Github Actions](https://github.com
 
 To deploy to a separate route (e.g. for testing / previewing changes) you can push to PaaS manually:
 
-```
+```sh
```

Comment:
pebble: do we need new details in the readme to describe the dockerfile and how to build/use it locally?

Reply:
Yes; that's still marked as to-do in [ENG-83](https://gds-cyber-and-engineering.atlassian.net/browse/ENG-83)